### ISSUE TITLE
Documentation of incompatibilities between prost & rddlsim?

### ISSUE THREAD

**Issue by fdouglis** [prost-planner/prost#125]
I note that if I try to test an RDDL setup with rddlsim before letting PROST fly, it complains about action-preconditions that PROST is happy with.  (It said the "proceed" action violated its expectations of what was true, though it runs fine under PROST.)  I wanted to run one of your canned examples using these intermediate levels directly in rddlsim, but both the wildfire2018 and push-your-luck resulted in overflowing Java's stack.  (this was with the 01 version of each of these.)  

I thought that PROST used the same rddlsim under the covers, so i'd expect them to be able to run the same things, but maybe not.  I'm wondering if this is a known issue or my misconfiguration, and if a known issue, are there any workarounds?  Is there a list of such differences someplace?  Thanks.

**Comment by thomaskeller79** [comment]
@fdouglis, could you please check if the error also occurs if you use the rddlsim version on the branch "ipc2018"? (Don't forget to run compile after switching to that branch.)

If this doesn't change anything, could you please share a minimal example for which the issue occurs?

**Comment by fdouglis** [comment]
I just checked out ippc2018 (extra p there but I assume that's what was meant) and compiled, then ran push-your-luck:

`./run rddl.sim.Simulator ../prost/testbed/benchmarks/push-your-luck-2018 rddl.policy.RandomBoolPolicy push-your-luck_inst_mdp__01`

Got the same stack overflow.   Should I expect that to work?

**Comment by thomaskeller79** [comment]
No, I've got the same problem. You can run push-your-luck with prost and the rddlsim competition server (rddl.competition.Server) though and it works fine, so I assume the problem is somewhere in rddl.policy.RandomBoolPolicy (which I have never used prior to our conversation). It's probably best to open an issue in the rddlsim repo for this.

**Comment by fdouglis** [comment]
Right, that was the gist of my initial post -- things that run with prost (including the server) hit problems with running rddlsim directly.  The stack overflowing is one example and it arises with your example (push-your-luck), but before that I hit a different issue where a constraint that ran fine with prost was hitting an exception in standalone.  

I can raise an issue in rddlsim -- thanks @thomaskeller79 ...
